src/content/reference/react/useContext.md

```diff
@@ -562,9 +563,9 @@ label {
 
 <Solution />
 
-#### Extracting providers to a component {/*extracting-providers-to-a-component*/}
+#### Mengekstrak penyedia ke komponen {/*extracting-providers-to-a-component*/}
 
-As your app grows, it is expected that you'll have a "pyramid" of contexts closer to the root of your app. There is nothing wrong with that. However, if you dislike the nesting aesthetically, you can extract the providers into a single component. In this example, `MyProviders` hides the "plumbing" and renders the children passed to it inside the necessary providers. Note that the `theme` and `setTheme` state is needed in `MyApp` itself, so `MyApp` still owns that piece of the state.
+Seiring dengan pertumbuhan aplikasi Anda, diharapkan Anda akan memiliki "piramida" konteks yang lebih dekat dengan akar aplikasi Anda. Tidak ada yang salah dengan hal itu. Namun, jika Anda tidak menyukai susunan tersebut secara estetika, Anda bisa mengekstrak penyedia ke dalam satu komponen. Dalam contoh ini, `MyProviders` menyembunyikan "pipa ledeng" dan membuat anak-anak yang diteruskan ke pipa ledeng tersebut di dalam penyedia yang diperlukan. Perhatikan bahwa *state* `theme` dan `setTheme` dibutuhkan di dalam `MyApp` itu sendiri, jadi `MyApp` masih memiliki bagian *state* tersebut.
```

Comment:
Because the original is `As your app grows`, not `Along with your application growth`.

Also need to confirm whether to stay to use `root` or translated into `akar`. It's not in Glossary yet. Discussion: https://github.com/reactjs/id.react.dev/discussions/337

`"plumbing"` as an idiom might be better translated into `"cara kerja"`, because `"pipa ledeng"` can cause a confusion in Indonesian.

`anak-anak` might be need more context as `props anak-anak`, because without the proper context, the sentence becomes ambiguous.

```suggestion
Seiring aplikasi Anda bertumbuh, diharapkan Anda akan memiliki "piramida" konteks yang lebih dekat dengan akar aplikasi Anda. Tidak ada yang salah dengan hal itu. Namun, jika Anda tidak menyukai susunan tersebut secara estetika, Anda bisa mengekstrak penyedia ke dalam satu komponen. Dalam contoh ini, `MyProviders` menyembunyikan "cara kerja" dan membuat anak-anak yang diteruskan ke hal tersebut di dalam penyedia yang diperlukan. Perhatikan bahwa *state* `theme` dan `setTheme` dibutuhkan di dalam `MyApp` itu sendiri, jadi `MyApp` masih memiliki bagian *state* tersebut.
```

Reply:
Bagaimana dengan "Seiring berkembangnya aplikasi Anda" @mhaidarhanif ?

Reply:
@r17x Yes, need to decide either:

1. Seiring berkembangnya aplikasi Anda
2. Seiring aplikasi Anda bertumbuh
3. Seiring aplikasi Anda berkembang

Reply:
The most difficult part remains about the `"plumbing"`.

"plumbing" as an idiom and in this context might be better translated into "cara kerja", because "pipa ledeng" can cause confusion in Indonesian.

Reply:
I believe we should use `tumbuh` as the direct literal translation. So, in this case, it should be "Seiring dengan bertumbuhnya aplikasi Anda".

Reply:
@zainfathoni @r17x Okay so these are the options:

1. Seiring dengan bertumbuhnya aplikasi Anda
2. Seiring bertumbuhnya aplikasi Anda
3. Seiring aplikasi Anda bertumbuh

Reply:
Alright "Seiring dengan bertumbuhnya aplikasi Anda"